src/components/DashboardsBar/NavigationMenu/__tests__/NavigationMenu.spec.js

```diff
@@ -86,5 +86,5 @@ test('renders a placeholder list item if no dashboards meet the filter criteria'
     expect(getByPlaceholderText('Search for a dashboard')).toHaveValue(
         filterStr
     )
-    expect(getByText('No dashboards found')).toBeVisible()
+    expect(getByText('No dashboards found for "xxxxxxxxxxxxx"')).toBeVisible()
```

Comment:
Perhaps implement with a template string, because we also do `toHaveValue(filterStr)`:
```suggestion
    expect(getByText(`No dashboards found for "${filterString}"`)).toBeVisible()
```